<launch>
    <!-- Launch ad_hb_msgs node -->
    <node pkg="ad_hb_msgs" type="ad_hb_msgs.py" name="ad_hb_msgs_node" output="screen" />

    <!-- Launch attitude_control node -->
    <node pkg="attitude_control" type="attitude_control.py" name="attitude_control_node">
        <param name="kp_elevator" value="10.0" />
        <param name="ki_elevator" value="0.00" />
        <param name="kd_elevator" value="10.0" />
        <param name="kp_rudder" value="3" />
        <param name="ki_rudder" value="0" />
        <param name="kd_rudder" value="3" />
        <param name="kp_prop" value="1.2" />
        <param name="ki_prop" value="0.3" />
        <param name="kd_prop" value="3" />
        <param name="kx" value="1.0" />
        <param name="ky" value="1.0" />
    </node>

    <!-- Launch auv_imp node -->
    <node pkg="auv_imp" type="auv_imp.py" name="auv_imp_node" output="screen" />
</launch>
